charts/platform-tenant/templates/components/apps/authorization-policy.yaml

```diff
@@ -1,27 +1,29 @@
 {{- $istio := .Values.components.istio -}}
-{{- if $istio.enabled -}}
+{{- $gateway := $istio.components.gateway -}}
+{{- if and $istio.enabled $gateway.enabled -}}
 {{ range .Values.components.apps }}
+{{- $app := . }}
 ---
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
-  name: {{ list .name $ | include "platform-tenant.format.name.shared" }}
+  name: {{ list $app.name $ | include "platform-tenant.format.name.shared" }}
   namespace: {{ list $ | include "platform-tenant.format.namespace" }}
   labels: {{- include "skyfjell.common.chartLabels" $ | nindent 4 }}
 spec:
   selector:
     matchLabels:
-      app: {{ .name }}
+      app: {{ $app.name }}
```

Comment:
We should always use k8s recommended labels for all of these selectors. At least name/instance,

```
app.kubernetes.io/name
app.kubernetes.io/instance
```